dbt_platform_helper/utils/validation.py

```diff
@@ -266,6 +266,25 @@ def iam_role_arn_regex(key):
     )
 
 
+def dbt_email_address_regex(key):
+    return Regex(
+        r"^[\w\-\.]+@(businessandtrade.gov.uk|digital.trade.gov.uk)$",
```

Comment:
I don't think you need to escape the . and hyphen in the character group, but the hyphen will probably need to come last to stop the regex parser from thinking it's a character range.
```suggestion
        r"^[\w.-]+@(businessandtrade.gov.uk|digital.trade.gov.uk)$",
```

Are we sure that these are the only characters used in DBT email addresses?

Reply:
yes, we only use these two.  Eveyones old address have been moved over to businessandtrade